kili/utils/bucket.py

```diff
@@ -5,17 +5,18 @@
 from urllib.parse import parse_qs, urlparse
 
 import requests
+from kili.authentication import KiliAuth
+from kili.graphql.operations.asset.queries import GQL_CREATE_UPLOAD_BUCKET_SIGNED_URLS
```

Comment:
kili import should not be there. This is probably because there has been a `pip install kili` in your environment